app/src/main/java/com/infomaniak/mail/ui/main/newMessage/AiPromptFragment.kt

```diff
@@ -95,18 +90,35 @@ class AiPromptFragment : Fragment() {
             R.color.aiBetaTagTextColor,
         )
 
-        prompt.showKeyboard()
+        prompt.post { prompt.showKeyboard() }
```

Comment:
Because else the keyboard doesn't show up when using the "modify" shortcut. This happens because the popup menu is still displayed and prevents the keyboard from opening if called directly